cmd/healthchecker/options/options.go

```diff
@@ -33,13 +33,15 @@ func NewHealthCheckerOptions() *HealthCheckerOptions {
 
 // HealthCheckerOptions are the options used to configure the health checker.
 type HealthCheckerOptions struct {
-	Component          string
-	SystemdService     string
-	EnableRepair       bool
-	CriCtlPath         string
-	CriSocketPath      string
-	CoolDownTime       time.Duration
-	HealthCheckTimeout time.Duration
+	Component                string
+	SystemdService           string
+	EnableRepair             bool
+	CriCtlPath               string
+	CriSocketPath            string
+	CoolDownTime             time.Duration
+	HealthCheckTimeout       time.Duration
+	LogPattern               string
```

Comment:
We should support multiple log patterns. Can we have a customized flag supporting this, e.g. https://play.golang.org/p/Ig5sm7jA14?

It could be something like this:
```
health-checker --log-pattern 10:aaaaaa --log-pattern 20:bbbbbb --log-pattern 15:ccccc
```

Reply:
I see. Yeah that can be helpful.

Is the idea in the above example <threshold count>:<pattern>?

Reply:
Yeah. If we do pattern:threshold_count, it is possible that the pattern may contain `:`, and make the parsing harder.

Reply:
Make sense. Changed.